app/views/hosts/_form.html.erb

```diff
@@ -4,7 +4,7 @@
 <%= render "hosts/progress" %>
 
 <% Taxonomy.as_taxonomy @organization , @location do %>
-
+<div id="host_form">
   <%= form_for @host, :url => (@host.new_record? ? hosts_path : host_path(:id => @host.id)), :html => {:data => {:id => @host.try(:id), :type_changed => !!@host.type_changed?, :submit => 'progress_bar'}} do |f| %>
```

Comment:
```suggestion
  <%= form_for @host, :url => (@host.new_record? ? hosts_path : host_path(:id => @host.id)), :html => {:id => 'host_form', :data => {:id => @host.try(:id), :type_changed => !!@host.type_changed?, :submit => 'progress_bar'}} do |f| %>
```

Reply:
You want only one space? :eyes:

Reply:
no, i want the id on the form instead of a wrapper div :)

Reply:
ah, it was too far xD I didn't see.
What about this?